02-inspect-file.md

```diff
@@ -169,7 +169,7 @@ Instead of simply counting the files that match the search criteria, save them t
 This is done with the redirection operator, `>`.
 
 ~~~ {.bash}
-gunzip -c counts-raw.txt.gz | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
+zcat counts-raw.txt.gz | cut -f11 | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
```

Comment:
I think it would be easiest if you just reverted that one commit and then pushed it:

```
git revert e103fb82612e1629a9c0db11a0ece205cf4c8c99
git push origin gh-pages
```

The PR will be automatically updated with this new commit that reverses that previous one.